app/src/main/java/dev/johnoreilly/fantasypremierleague/presentation/fixtures/FixturesViewModel.kt

```diff
@@ -0,0 +1,16 @@
+package dev.johnoreilly.fantasypremierleague.presentation.fixtures
+
+import androidx.lifecycle.ViewModel
+import dev.johnoreilly.common.data.repository.FantasyPremierLeagueRepository
+import kotlinx.coroutines.flow.StateFlow
+import org.koin.java.KoinJavaComponent.inject
+
+class FixturesViewModel: ViewModel() {
+
+    private val repository: FantasyPremierLeagueRepository by inject(FantasyPremierLeagueRepository::class.java)
```

Comment:
you can pass `repository` in the constructor (as `FantasyPremierLeagueViewModel` is (the Koin `get()` will automatically inject it)